terraform/modules/cloudfront_distributions/main.tf

```
@@ -23,7 +23,10 @@ module "delta_website_waf" {
   log_group_suffix  = "delta-website-${var.environment}"
   per_ip_rate_limit = var.waf_per_ip_rate_limit
   # Orbeon triggers this rule
-  excluded_rules                 = ["CrossSiteScripting_BODY", "GenericLFI_BODY"]
+  excluded_rules = ["CrossSiteScripting_BODY",
+    "GenericLFI_BODY",
+    # To allow long query strings - fix for MSD-80376
+  "SizeRestrictions_QUERYSTRING"]
```

I think Hugh meant formatted like this, lgtm anyway

```suggestion
  excluded_rules = [
    "CrossSiteScripting_BODY",     # Sometimes blocks File uploads
    "GenericLFI_BODY",             # Sometimes blocks Excel uploads
   "SizeRestrictions_QUERYSTRING", # To allow long query strings - fix for MSD-80376
 ]
```